eln/SampleFactory.js

```diff
@@ -0,0 +1,7 @@
+import Sample from './ExtSample';
+
+export async function SampleFactory(options = {}) {
```

Comment:
I'd export `extSample` (lower case and distinct from possibly later added factory function for `Sample` with roc)